xpdtools/pipelines/calibration.py

```diff
@@ -0,0 +1,46 @@
+from streamz import Stream
+from xpdtools.calib import img_calibration
+from xpdtools.tools import load_geo, generate_map_bin, map_to_binner
+
+
+def cali():
```

Comment:
maybe call it calibration?  is the word-shortening worth it in the long run?

Also, do we need a docstring here?